source/installation/windows.txt

```diff
@@ -0,0 +1,114 @@
+.. _cpp-installation-windows:
+
+=======================
+Installation on Windows
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+Step 1: Choose a C++17 polyfill
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, :ref:`choose a C++17 polyfill library <cpp-polyfill-selection>`.
+
+Step 2: Download the latest version of the mongocxx driver
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The most reliable starting point for building the mongocxx driver is the latest
+release tarball.
+
+The `mongocxx releases <https://github.com/mongodb/mongo-cxx-driver/releases>`__
+page will have links to the release tarball for the version you wish you install.  For
+example, to download version 3.10.0:
+
+.. code-block:: bash
+
+   curl -OL https://github.com/mongodb/mongo-cxx-driver/releases/download/r3.10.0/mongo-cxx-driver-r3.10.0.tar.gz
+   tar -xzf mongo-cxx-driver-r3.10.0.tar.gz
+   cd mongo-cxx-driver-r3.10.0/build
+
+Make sure you change to the ``build`` directory of whatever source tree you
+obtain.
+
+Step 3: Configure the driver
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On Windows, the C++ driver is configured as follows (adjusting the path of the CMake executable as appropriate to your system):
+
+.. code-block:: bash
+
+   'C:\Program Files (x86)\CMake\bin\cmake.exe' .. \
+        -G "Visual Studio 14 2015" -A "x64"        \
+       -DBOOST_ROOT=C:\local\boost_1_60_0          \
+       -DCMAKE_INSTALL_PREFIX=C:\mongo-cxx-driver
+
+The example above assumes:
+
+- Boost is found in ``C:\local\boost_1_60_0``.
+- ``mongocxx`` is to be installed into ``C:\mongo-cxx-driver``.
+
+To build with Visual Studio 2017 without a C++17 polyfill, configure as follows:
+
+.. code-block:: bash
+
+   'C:\Program Files (x86)\CMake\bin\cmake.exe' .. \
+       -G "Visual Studio 15 2017" -A "x64"         \
+       -DCMAKE_CXX_STANDARD=17                     \
+       -DCMAKE_INSTALL_PREFIX=C:\mongo-cxx-driver  \
+
+Configuring with ``mongocxx`` 3.7.0 and Older
+`````````````````````````````````````````````
+
+To build versions 3.7.0 and older without a C++17 polyfill, it is necessary to configure with additional options:
```

Comment:
Add newline? Rendering does not include bullets [on staging](https://docs-mongodbcom-staging.corp.mongodb.com/cpp-driver/docsworker-xlarge/content-formatting/installation/windows/):

![Screen Shot 2024-03-12 at 3 17 01 PM](https://github.com/mongodb/docs-cpp/assets/992997/e5bad6e1-0c20-4860-b22a-574d8996bf98)